config/locales/sv.yml

```diff
@@ -1054,7 +1057,19 @@ sv:
         message_text:
           fee_is_unpaid: Observera att ditt företags H-märkespremie inte är betald.
           how_to_pay_fee: Betala den ett år till via länken nedan. Detta gör att du fortsätter synas på kartan och fortsätter ha ett H-märkt företag.
-          company_link_login_msg: " (du kan behöva logga in först innan du ser knappen för att betala)."
+          company_link_login_msg: &co_link_login " (du kan behöva logga in först innan du ser knappen för att betala)."
+
+      co_info_incomplete:
+        subject: Var god färdigställ din företagssida
+        greeting: *email_greeting
+        message_text:
+          complete_the_info: "Du behöver färdigställa informationen om %{co_identifier}. Om du inte gör det syns du inte i sökningar. Saknar du adress syns du heller inte på kartan."
+          company_needs_name: "företag: %{co_number}"
+          what_is_missing: "Här är vad som saknas:"
+          co_name_missing: "det finns inget namn för företaget"
+          co_region_missing: "regionen för företaget tillhandahålls inte"
```

Comment:
Postadress

Reply:
We are specifically checking that the **region** is not nil.  They might have other parts of the address in there, but if they left out the Region, we call it "incomplete".

So - I'm just checking that the Swedish wording reflects that it's the _region_.

Is "postadress" really what you want there?

Reply:
Yes, they dont need to know what we are checking :) 
and I'm fine with them not being displayed on the map if they chose not to be. But I'm not fine with them not having a postadress :D